linker/Linker/Annotations.cs

```diff
@@ -316,7 +322,7 @@ public void CloseSymbolReader (AssemblyDefinition assembly)
 			return slots;
 		}
 
-		public void AddDependency (object o)
+		public void AddDependency (object o, bool mark=false)
```

Comment:
What does mark mean in this case?

Reply:
The mark value indicates whether the dependency was added when o was marked by the linker.

Reply:
could you rename the parameter to be `marked` then ?

Reply:
ok, updated